cfme/common/provider_views.py

```diff
@@ -0,0 +1,277 @@
+# -*- coding: utf-8 -*-
+from widgetastic.widget import View, Text
+
+from cfme import BaseLoggedInPage
+from widgetastic_patternfly import Dropdown, BootstrapSelect
+from widgetastic_manageiq import (BreadCrumb,
```

Comment:
mfalesni states that both libs will be external ones soon. so, I've moved both imports to external libs.